pkg/apis/network/v1/egressrouter_types.go

```diff
@@ -0,0 +1,273 @@
+package v1
```

Comment:
wrong repo. This must go to openshift/api.

Reply:
This is a CRD that the CNO consumes - do we need to put it in openshift/api?

Reply:
there was a discussion on this and as long as we document this properly it should be fine right ? Is there a strict rule for mapping all types to `openshift/api`? We use this types.go to auto-generate the yaml file, hence asking.